includes/class-wc-payments-payment-method-messaging-element.php

```diff
@@ -48,10 +48,27 @@ public function __construct( WC_Payments_Account $account, $gateway ) {
 	 */
 	public function init(): string {
 		global $product;
-		$price           = $product->get_price();
-		$currency_code   = get_woocommerce_currency();
-		$store_country   = WC()->countries->get_base_country();
-		$billing_country = WC()->customer->get_billing_country();
+		$currency_code      = get_woocommerce_currency();
+		$store_country      = WC()->countries->get_base_country();
+		$billing_country    = WC()->customer->get_billing_country();
+		$base_product_price = WC_Payments_Utils::prepare_amount( $product->get_price(), $currency_code );
```

Comment:
Can we probably omit this variable and inject the stored value where it's being used?

Reply:
Ah. I missed removing that in an earlier iteration. I have injected the value directly at the only place it is now needed. 

ef11e5a